tests/ui-toml/undocumented_unsafe_blocks/undocumented_unsafe_blocks.rs

```diff
@@ -0,0 +1,12 @@
+#![deny(clippy::undocumented_unsafe_blocks)]
+#![allow(clippy::missing_safety_doc)]
+
+fn main() {
+    // Safety: A safety comment
+    let _some_variable_with_a_very_long_name_to_break_the_line =
+        unsafe { a_function_with_a_very_long_name_to_break_the_line() };
```

Comment:
I think this could benefit from an extra test or two; namely, making sure static/consts work as expected as well

Reply:
Done! https://github.com/rust-lang/rust-clippy/pull/10886/commits/d01f49aad11eb505bbd9bb34024cc5e46b875350

Thanks for sugesting this! In fact, the result was not what we expected for static/const. I had to make another change, as well, for those two cases.